### PR TITLE
Witness: cta style

### PR DESCRIPTION
**Pull request by stephanfowler**
(also added at article bottom)

witness

### CODE REVIEW

File: static/src/stylesheets/module/external/_witness.scss [hunk 1]
```diff
@@ -69,46 +69,36 @@
...
+    white-space: nowrap;
+    margin-right: 0;
 }
+
 .witness-logo {
     display: inline-block;
-    color: #247eb3;
-    @include font($f-serif-text, 700, 16, 16);
+    color: #ffffff;
+    font-weight: 700;
```
Comment (rich-nguyen) [inline static/src/stylesheets/module/external/_witness.scss]: mixin?

**Comment by rich-nguyen** [comment]
:+1:

**Comment by chriswilk** [comment]
Looking good @stephanfowler ! 

Just a couple of things that you may have already discussed with the designers and decided change amend. The apps team have followed this.

image

- Missing plus (+) icon. This was here to give visual indication that you as the reader would be adding the to the story.
- Different font colour. The original design was using the witness brand blue. Here is the logo

image

Thanks

File: static/src/stylesheets/module/external/_witness.scss [hunk 2]
```diff
@@ -69,54 +69,38 @@
...
-    @include fs-textSans(3);
-    float: left;
-    line-height: 15px;
-    padding: $gs-baseline/2 $gs-gutter/2 0;
-    margin-left: 50px;
 
-    .witness-cta__logo {
-        display: block;
-    }
+.button--witness {
```
Comment (uplne) [inline static/src/stylesheets/module/external/_witness.scss]: From the logic of the BEM naming convention I would look for the '.button--witness' modification in the 'button.scss' file.

Ideal solution would be to use some generic button modification we are already using on the page ('.button--large') so we wouldn't be creating another version of the button to the code base. But this is going down to UX and design.

If we want to create some specific version of the button just for some unique object ('.witness' object in this case) I would go for something like '.witness__button'. From this it makes perfect sense that button is using object '.button' but also specific element styles of the object '.witness'.

File: article/app/views/fragments/witnessCallToAction.scala.html [hunk 3]
```diff
@@ -7,9 +7,8 @@
         @defining(s"https://witness.theguardian.com/assignment/$assignmentId#contribute"){ assignmentUrl =>
             <a href="@LinkTo{@assignmentUrl}" data-link-name="witness cta">
                 <div class="witness-cta-wrapper">
-                    <i class="i i-ee-add-button witness-cta__icon"></i>
                     <div class="witness-cta">
-                        <div class="witness-cta__body">
+                        <div class="button button--large button--primary button--witness">
                             Contribute with
                             <span class="witness-logo witness-cta__logo">Guardian <span class="witness-logo__witness">Witness</span></span>
```
Comment (uplne) [inline article/app/views/fragments/witnessCallToAction.scala.html]: Is '.witness-cta__logo' being used? It seems to be removed from _witness.scss

File: static/src/stylesheets/module/external/_witness.scss (hunk 2 above)
Comment (davidrapson) [inline static/src/stylesheets/module/external/_witness.scss]: (:clap: seconding what @uplne said)

Comment (sammorrisdesign) [inline static/src/stylesheets/module/external/_witness.scss]: Unfortunately `.button` comes in through a bower component, so we can't include it there.

Comment (sammorrisdesign) [inline static/src/stylesheets/module/external/_witness.scss]: Also there's this nonsense going on... https://github.com/guardian/frontend/blob/witness-style/static/src/stylesheets/module/external/_witness.scss#L2

Comment (uplne) [inline static/src/stylesheets/module/external/_witness.scss]: Yes. And the thing is that we shouldn't anyway as this button is a very specific object for only .com project and button component should be for a button abstraction which can be used across the whole organisation.

That's one of the reasons why I would recommend not using '.button--witness' but '.witness__button' thus making it **element of the witness object instead** of **modification of the button object**.

I am happy to explain it more in person if you like.

**Comment by sammorrisdesign** [comment]
Updated:
screen shot 2015-06-08 at 12 15 47

**Pull request merged**